<launch>
	<!-- run the position_control.cpp -->

	<node pkg="px4_command" type="px4_traj_controller" name="px4_traj_controller" output="screen">
		
	<param name="switch_ude" type="int" value="0" />

	<rosparam command="load" file="$(find px4_command)/config/Parameter_for_control.yaml" />

	</node>
</launch>

